<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Input formats and creation of a REVOLVER cohort • revolver</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Input formats and creation of a REVOLVER cohort">
<meta property="og:description" content="revolver">
<meta property="og:image" content="caravagnalab.github.io/revolver/logo.png">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">revolver</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.3.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/revolver.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/Cross_CRC_MSeq.html">Analysis of colorectal adenocarcinomas</a>
    </li>
    <li>
      <a href="../articles/FAQ.html">Frequently Asked Questions</a>
    </li>
    <li>
      <a href="../articles/Getters.html">Getter functions</a>
    </li>
    <li>
      <a href="../articles/Input_formats.html">Input formats and creation of a REVOLVER cohort</a>
    </li>
    <li>
      <a href="../articles/Plotting_functions.html">Plotting functions</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/caravagn/revolver/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Input formats and creation of a REVOLVER cohort</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/caravagn/revolver/blob/HEAD/vignettes/Input_formats.Rmd" class="external-link"><code>vignettes/Input_formats.Rmd</code></a></small>
      <div class="hidden name"><code>Input_formats.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="input-format-for-revolver">Input format for REVOLVER<a class="anchor" aria-label="anchor" href="#input-format-for-revolver"></a>
</h2>
<p>Data from a cohort of patients can be represented as a <em>dataframe</em> with at least 7 columns, where every row represents one genomic alteration annotated for the analysis.</p>
<table class="table">
<colgroup>
<col width="33%">
<col width="33%">
<col width="33%">
</colgroup>
<thead><tr class="header">
<th>Field</th>
<th>Type</th>
<th>Description</th>
</tr></thead>
<tbody>
<tr class="odd">
<td><code>Misc</code></td>
<td>string</td>
<td><em>Customary annotation which is never used to analyse the data, but that might be good to carry around</em></td>
</tr>
<tr class="even">
<td><code>patientID</code></td>
<td>string</td>
<td><em>A patient identifier</em></td>
</tr>
<tr class="odd">
<td><code>variantID</code></td>
<td>string</td>
<td><em>An alteration identifier</em></td>
</tr>
<tr class="even">
<td><code>cluster</code></td>
<td>string</td>
<td><em>Group ID (eg., a clone, with CCF data)</em></td>
</tr>
<tr class="odd">
<td><code>is.driver</code></td>
<td>logical</td>
<td>TRUE <em>if the alteration is a driver to correlate when repeated trajectores are extracted</em>
</td>
</tr>
<tr class="even">
<td><code>is.clonal</code></td>
<td>logical</td>
<td>TRUE <em>if the alteration belongs to the group of clonal (truncal) events; there should only one such clonal group per patient</em>
</td>
</tr>
<tr class="odd">
<td><code>CCF</code></td>
<td>string</td>
<td><em>A parsable format for storage of input CCFs or binary data (see below)</em></td>
</tr>
</tbody>
</table>
<p>All identifiers cannot contain spaces or dash/ hyphen (<code>-</code>) symbols, and must be unique:</p>
<ul>
<li><p>the <code>patientID</code> must be unique across patients;</p></li>
<li><p>the <code>variantID</code> for driver events (those with <code>is.driver = TRUE</code>) must be shared across all patients that have the same driver event, and cannot appear multiple times in the data of a single patient. Notice that you should not annotate an alteration as driver if it is not found in multiple patients.</p></li>
<li><p>the <code>variantID</code> for non-driver events (those with <code>is.driver = FALSE</code>) is in practice never used and therefore can be set freely.</p></li>
</ul>
<p>Notice that the input dataframe has the same structure for both CCF and binary input data.</p>
<div class="section level3">
<h3 id="supported-alterations">Supported alterations<a class="anchor" aria-label="anchor" href="#supported-alterations"></a>
</h3>
<p>The scope of supported alterations is broad, and include any SNV, larger chromosomal re-arrangment or other covariate that can be encoded in CCF or binary format.</p>
<p>The <code>variantID</code> field of driver alterations (<code>is.driver=TRUE</code>) will be matched to correlate trajectories in multiple patients, and can be you whatever you find more suitable for your analysis, for instance:</p>
<ul>
<li><p>a Hugo_Symbol (<code>BRAF</code>)</p></li>
<li><p>a name for a well-known SNV (<code>BRAF_600E</code>)</p></li>
<li><p>a reference to some cytoband (<code>3q26.32</code>)</p></li>
<li><p>your custom annotation (<code>MyFavoritePathway</code>).</p></li>
</ul>
<p>Alterations are also associated to groups (via <code>cluster</code>), which constitute the nodes of the computed trees. A group can have 0 drivers annotated, but every patient should have at least one driver to be analyzed with <code>REVOLVER</code>.</p>
<p>See also the <a href="https://caravagn.github.io/revolver/articles/FAQ.html" class="external-link">Frequently Asked Questions</a> if you are interested in modelling parallel evolution.</p>
</div>
<div class="section level3">
<h3 id="input-values-for-ccf-or-binary-datasets">Input values for CCF or binary datasets<a class="anchor" aria-label="anchor" href="#input-values-for-ccf-or-binary-datasets"></a>
</h3>
<p>Field <code>CCF</code> represents data from both types of <a href="https://caravagn.github.io/revolver/articles/FAQ.html" class="external-link">supported datasets</a>:</p>
<ul>
<li><p>real-valued CCF values (in [0, 1]),</p></li>
<li><p>or input binary values (either 0 or 1).</p></li>
</ul>
<p>Since patients can have different number of samples/ regions associated, <code>CCF</code> is a general string.</p>
<p>The format that we propose is simple, and easy to parse it:</p>
<pre><code><span class="co"># Example string that reports a CCF value 0.86 in region R1, 1 in R2 etc. </span>
<span class="va">R1</span><span class="op">:</span><span class="fl">0.86</span>;<span class="va">R2</span><span class="op">:</span><span class="fl">1</span>;<span class="va">R3</span><span class="op">:</span><span class="fl">1</span>

<span class="co"># Similarly, in binary presence/ absence format ... </span>
<span class="va">R1</span><span class="op">:</span><span class="fl">0</span>;<span class="va">R2</span><span class="op">:</span><span class="fl">1</span>;<span class="va">R3</span><span class="op">:</span><span class="fl">1</span></code></pre>
<p>If you use this format, <code>REVOLVER</code> provides you a <a href="https://caravagn.github.io/revolver/reference/CCF_parser.html" class="external-link">parsing function</a> that you can pass to the <a href="https://caravagn.github.io/revolver/reference/revolver_cohort.html" class="external-link">cohort construction function</a></p>
</div>
<div class="section level3">
<h3 id="example-data">Example data<a class="anchor" aria-label="anchor" href="#example-data"></a>
</h3>
<p>An example cohort is released in the <code>evoverse.datasets</code> <a href="https://github.com/caravagn/evoverse.datasets" class="external-link">R package</a>, which also provides a <code>REVOLVER</code> objectwith the final analysis of the input cohort.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Load the package and runs available_data() to print what is available</span>
<span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/caravagn/evoverse.datasets" class="external-link">evoverse.datasets</a></span><span class="op">)</span>
<span class="co">#&gt; Warning: package 'crayon' was built under R version 4.0.5</span>

<span class="co"># Load an example dataset - the TRACERx cohort</span>
<span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="st">"TRACERx_NEJM_2017"</span>, package <span class="op">=</span> <span class="st">"evoverse.datasets"</span><span class="op">)</span>

<span class="co"># This is a dataset that follows the specifications given above</span>
<span class="fu"><a href="https://rdrr.io/r/base/print.html" class="external-link">print</a></span><span class="op">(</span><span class="va">TRACERx_NEJM_2017</span><span class="op">)</span>
<span class="co">#&gt; <span style="color: #949494;"># A tibble: 65,421 × 7</span></span>
<span class="co">#&gt;    Misc                    patientID variantID cluster is.driver is.clonal CCF  </span>
<span class="co">#&gt;    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>                   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>   <span style="color: #949494; font-style: italic;">&lt;lgl&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;lgl&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span></span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span> CRUK0001:7:47971467:C   CRUK0001  PKD1L1    1       FALSE     FALSE     R1:0…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span> CRUK0001:21:47754753:C  CRUK0001  PCNT      1       FALSE     FALSE     R1:0…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span> CRUK0001:22:37331483:G  CRUK0001  CSF2RB    1       FALSE     FALSE     R1:0…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span> CRUK0001:10:102824636:G CRUK0001  KAZALD1   1       FALSE     FALSE     R1:0…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span> CRUK0001:7:81978881:C   CRUK0001  CACNA2D1  1       FALSE     FALSE     R1:0…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span> CRUK0001:1:201843493:G  CRUK0001  IPO9      1       FALSE     FALSE     R1:0…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span> CRUK0001:3:167402119:G  CRUK0001  PDCD10    1       FALSE     FALSE     R1:0…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span> CRUK0001:7:134650062:G  CRUK0001  CALD1     1       FALSE     FALSE     R1:0…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span> CRUK0001:17:76989965:C  CRUK0001  CANT1     1       FALSE     FALSE     R1:0…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;">10</span> CRUK0001:16:1662386:G   CRUK0001  IFT140    1       FALSE     FALSE     R1:0…</span>
<span class="co">#&gt; <span style="color: #949494;"># … with 65,411 more rows</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="cohort-construction">Cohort construction<a class="anchor" aria-label="anchor" href="#cohort-construction"></a>
</h2>
<p>Once you have a dataframe/ tibble in the required format, you can use a <a href="https://caravagn.github.io/revolver/reference/revolver_cohort.html" class="external-link">cohort construction function</a>.</p>
<p>In this case, to make it faster we only use 3 patients from the released cohort.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">require</a></span><span class="op">(</span><span class="va"><a href="https://tidyverse.tidyverse.org" class="external-link">tidyverse</a></span><span class="op">)</span>
<span class="co">#&gt; Loading required package: tidyverse</span>
<span class="co">#&gt; ── <span style="font-weight: bold;">Attaching packages</span> ─────────────────────────────────────── tidyverse 1.3.1 ──</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">ggplot2</span> 3.3.6     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">purrr  </span> 0.3.4</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">tibble </span> 3.1.7     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">dplyr  </span> 1.0.9</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">tidyr  </span> 1.2.0     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">stringr</span> 1.4.0</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">readr  </span> 2.1.2     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">forcats</span> 0.5.1</span>
<span class="co">#&gt; Warning: package 'tidyr' was built under R version 4.0.5</span>
<span class="co">#&gt; Warning: package 'readr' was built under R version 4.0.5</span>
<span class="co">#&gt; ── <span style="font-weight: bold;">Conflicts</span> ────────────────────────────────────────── tidyverse_conflicts() ──</span>
<span class="co">#&gt; <span style="color: #BB0000;">✖</span> <span style="color: #0000BB;">ggplot2</span>::<span style="color: #00BB00;">%+%()</span>  masks <span style="color: #0000BB;">crayon</span>::%+%()</span>
<span class="co">#&gt; <span style="color: #BB0000;">✖</span> <span style="color: #0000BB;">dplyr</span>::<span style="color: #00BB00;">filter()</span> masks <span style="color: #0000BB;">stats</span>::filter()</span>
<span class="co">#&gt; <span style="color: #BB0000;">✖</span> <span style="color: #0000BB;">dplyr</span>::<span style="color: #00BB00;">lag()</span>    masks <span style="color: #0000BB;">stats</span>::lag()</span>
<span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">require</a></span><span class="op">(</span><span class="va"><a href="https://github.com/caravagn/revolver" class="external-link">revolver</a></span><span class="op">)</span>
<span class="co">#&gt; Loading required package: revolver</span>
<span class="co">#&gt; Warning: replacing previous import 'cli::num_ansi_colors' by</span>
<span class="co">#&gt; 'crayon::num_ansi_colors' when loading 'easypar'</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> Loading <span style="color: #00BB00;">ctree</span>, <span style="font-style: italic;">'Clone trees in cancer'</span>. Support : <span style="color: #0000BB; font-style: italic;">&lt;https://caravagn.github.io/ctree/&gt;</span></span>
<span class="co">#&gt; Warning: replacing previous import 'dendextend::theme_dendro' by</span>
<span class="co">#&gt; 'ggdendro::theme_dendro' when loading 'revolver'</span>
<span class="co">#&gt; Warning: replacing previous import 'dendextend::rotate' by 'ggpubr::rotate' when</span>
<span class="co">#&gt; loading 'revolver'</span>
<span class="co">#&gt; Warning: replacing previous import 'igraph::%s%' by 'matrixcalc::%s%' when</span>
<span class="co">#&gt; loading 'mtree'</span>
<span class="co">#&gt; Warning: replacing previous import 'igraph::groups' by 'tidygraph::groups' when</span>
<span class="co">#&gt; loading 'mtree'</span>
<span class="co">#&gt; Warning: package 'tidygraph' was built under R version 4.0.5</span>
<span class="co">#&gt; Warning: package 'igraph' was built under R version 4.0.5</span>
<span class="co">#&gt; Warning: package 'RColorBrewer' was built under R version 4.0.5</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> Loading <span style="color: #00BB00;">mtree</span>, <span style="font-style: italic;">'Mutation trees in cancer'</span>. Support : <span style="color: #0000BB; font-style: italic;">&lt;https://caravagn.github.io/mtree/&gt;</span></span>
<span class="co">#&gt; Warning: replacing previous import 'ctree::plot_information_transfer' by</span>
<span class="co">#&gt; 'mtree::plot_information_transfer' when loading 'revolver'</span>
<span class="co">#&gt; Warning: replacing previous import 'ctree::plot_clone_size' by</span>
<span class="co">#&gt; 'mtree::plot_clone_size' when loading 'revolver'</span>
<span class="co">#&gt; Warning: replacing previous import 'ctree::plot_icon' by 'mtree::plot_icon' when</span>
<span class="co">#&gt; loading 'revolver'</span>
<span class="co">#&gt; Warning: replacing previous import 'igraph::groups' by 'tidygraph::groups' when</span>
<span class="co">#&gt; loading 'revolver'</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> Loading <span style="color: #00BB00;">revolver</span>, <span style="font-style: italic;">'Repeated Evolution in Cancer'</span>. Support : <span style="color: #0000BB; font-style: italic;">&lt;https://caravagn.github.io/revolver/&gt;</span></span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Attaching package: 'revolver'</span>
<span class="co">#&gt; The following object is masked from 'package:igraph':</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;     plot_dendrogram</span>

<span class="va">subset_data</span> <span class="op">=</span> <span class="va">TRACERx_NEJM_2017</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span>
  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">patientID</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"CRUK0001"</span>, <span class="st">"CRUK0002"</span>, <span class="st">"CRUK0003"</span><span class="op">)</span><span class="op">)</span>

<span class="co"># Create a cohort</span>
<span class="va">my_cohort</span> <span class="op">=</span> <span class="fu"><a href="../reference/revolver_cohort.html">revolver_cohort</a></span><span class="op">(</span>
  <span class="va">subset_data</span>, 
  CCF_parser <span class="op">=</span> <span class="va">CCF_parser</span>,
  ONLY.DRIVER <span class="op">=</span> <span class="cn">FALSE</span>, 
  MIN.CLUSTER.SIZE <span class="op">=</span> <span class="fl">0</span>,
  annotation <span class="op">=</span> <span class="st">"My small REVOLVER dataset"</span>
  <span class="op">)</span>
<span class="co">#&gt; <span style="color: #BBBBBB; background-color: #0000BB;"> [ REVOLVER ~ Cohort constructor ] </span></span>
<span class="co"><span style="color: #BBBBBB; background-color: #0000BB;">#&gt; </span></span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Using <span style="font-weight: bold;">only driver</span> mutations.</span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Rejecting clusters with less than <span style="font-weight: bold;">0</span> mutations.</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">REVOLVER input data</span> <span style="color: #00BBBB;">─────────────────────────────────────────────────────────</span></span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #949494;"># A tibble: 2,710 × 9</span></span>
<span class="co">#&gt;    Misc              patientID variantID cluster is.driver is.clonal CCF   id   </span>
<span class="co">#&gt;    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>             <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>   <span style="color: #949494; font-style: italic;">&lt;lgl&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;lgl&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span></span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span> CRUK0001:7:47971… CRUK0001  PKD1L1    1       FALSE     FALSE     R1:0… __mu…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span> CRUK0001:21:4775… CRUK0001  PCNT      1       FALSE     FALSE     R1:0… __mu…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span> CRUK0001:22:3733… CRUK0001  CSF2RB    1       FALSE     FALSE     R1:0… __mu…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span> CRUK0001:10:1028… CRUK0001  KAZALD1   1       FALSE     FALSE     R1:0… __mu…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span> CRUK0001:7:81978… CRUK0001  CACNA2D1  1       FALSE     FALSE     R1:0… __mu…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span> CRUK0001:1:20184… CRUK0001  IPO9      1       FALSE     FALSE     R1:0… __mu…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span> CRUK0001:3:16740… CRUK0001  PDCD10    1       FALSE     FALSE     R1:0… __mu…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span> CRUK0001:7:13465… CRUK0001  CALD1     1       FALSE     FALSE     R1:0… __mu…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span> CRUK0001:17:7698… CRUK0001  CANT1     1       FALSE     FALSE     R1:0… __mu…</span>
<span class="co">#&gt; <span style="color: #BCBCBC;">10</span> CRUK0001:16:1662… CRUK0001  IFT140    1       FALSE     FALSE     R1:0… __mu…</span>
<span class="co">#&gt; <span style="color: #949494;"># … with 2,700 more rows, and 1 more variable: cluster_size &lt;int&gt;</span></span>
<span class="co">#&gt; </span>
<span class="co">#&gt; ── Preprocessing data (this may take some time)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; ...</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">Extracting clones table</span> <span style="color: #00BBBB;">─────────────────────────────────────────────────────</span></span>
<span class="co">#&gt; → <span style="color: #00BB00;">CRUK0001</span> : 2100 entries, 11 clone(s).</span>
<span class="co">#&gt; → <span style="color: #00BB00;">CRUK0002</span> : 280 entries, 7 clone(s).</span>
<span class="co">#&gt; → <span style="color: #00BB00;">CRUK0003</span> : 330 entries, 7 clone(s).</span>

<span class="co"># This is now a REVOLVER cohort with S3 methods available</span>
<span class="fu"><a href="https://rdrr.io/r/base/print.html" class="external-link">print</a></span><span class="op">(</span><span class="va">my_cohort</span><span class="op">)</span>
<span class="co">#&gt; <span style="color: #BBBBBB; background-color: #0000BB;"> [ REVOLVER - Repeated Evolution in Cancer ] </span></span>
<span class="co"><span style="color: #BBBBBB; background-color: #0000BB;">#&gt; </span></span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; Dataset :</span> My small REVOLVER dataset </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; Cohort  :</span> 3 patients, 2710 variants and 16 driver events. </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; </span></span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; Trees per patient    :</span> <span style="color: #BB0000;">NO</span> </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; Fit via TL           :</span> <span style="color: #BB0000;">NO</span> </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; REVOLVER clustering  :</span> <span style="color: #BB0000;">NO</span> </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; Jackknife statistics :</span> <span style="color: #BB0000;">NO</span> </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; </span> </span>
<span class="co">#&gt; For summary statistics see `?Stats_*(x)` with * = {cohort, drivers, trees, fits, clusters, ...} </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; </span></span>
<span class="co">#&gt; <span style="color: #000000;">     ┌──────────────────────────────────────────────────────────────────────┐</span></span>
<span class="co">#&gt;      <span style="color: #000000;">│</span><span style="background-color: #FFAF00;">                                                                      </span><span style="color: #000000;">│</span></span>
<span class="co">#&gt;      <span style="color: #000000;">│</span><span style="background-color: #FFAF00;">   </span><span style="color: #000000; background-color: #FFAF00;">WARNING - Driver variantIDs occuring only once could be removed.</span><span style="background-color: #FFAF00;">   </span><span style="color: #000000;">│</span></span>
<span class="co">#&gt;      <span style="color: #000000;">│</span><span style="background-color: #FFAF00;">                                                                      </span><span style="color: #000000;">│</span></span>
<span class="co">#&gt; <span style="color: #000000;">     └──────────────────────────────────────────────────────────────────────┘</span></span>
<span class="co">#&gt; <span style="color: #949494;"># A tibble: 14 × 7</span></span>
<span class="co">#&gt;    variantID numClonal p_clonal numSubclonal p_subclonal N_tot p_tot</span>
<span class="co">#&gt;    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>         <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>    <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span> TP53              1    0.333            0       0         1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span> MGA               1    0.333            0       0         1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span> WRN               1    0.333            0       0         1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span> MET               1    0.333            0       0         1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span> TERT              1    0.333            0       0         1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span> PIK3CA            1    0.333            0       0         1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span> CDKN2A            1    0.333            0       0         1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span> ARHGAP35          0    0                1       0.333     1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span> PASK              0    0                1       0.333     1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;">10</span> RB1               0    0                1       0.333     1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;">11</span> IKZF1             0    0                1       0.333     1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;">12</span> KRAS              0    0                1       0.333     1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;">13</span> EP300             0    0                1       0.333     1 0.333</span>
<span class="co">#&gt; <span style="color: #BCBCBC;">14</span> CTNNB1            0    0                1       0.333     1 0.333</span>
<span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">my_cohort</span><span class="op">)</span></code></pre></div>
<p><img src="Input_formats_files/figure-html/unnamed-chunk-3-1.png" width="700"></p>
<p>The cohort is now ready to be inspected and analysed (note that since we only used 3 patients a warning is raised to mention that the cohort contains drivers annotated which are not really recurrent and that, therefore, should be removed; if you construct the cohort object with the full set of 99 patients that warning will vanish).</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Load a full cohort - the TRACERx cohort</span>
<span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="st">"TRACERx_NEJM_2017_REVOLVER"</span>, package <span class="op">=</span> <span class="st">"evoverse.datasets"</span><span class="op">)</span>

<span class="co"># We can use S3 object functions to retrieve simple information about the cohort.</span>
<span class="co"># The `print` functions runs also the `revolver_check_cohort` function which </span>
<span class="co"># tells us that some patient have only 1 clone with drivers, and therefore they</span>
<span class="co"># can just be expanded.</span>
<span class="fu"><a href="https://rdrr.io/r/base/print.html" class="external-link">print</a></span><span class="op">(</span><span class="va">TRACERx_NEJM_2017_REVOLVER</span><span class="op">)</span>
<span class="co">#&gt; <span style="color: #BBBBBB; background-color: #0000BB;"> [ REVOLVER - Repeated Evolution in Cancer ] </span></span>
<span class="co"><span style="color: #BBBBBB; background-color: #0000BB;">#&gt; </span></span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; Dataset :</span> TRACERx NEJM 2017 </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; Cohort  :</span> 99 patients, 450 variants and 79 driver events. </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; </span></span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; Trees per patient    :</span> <span style="color: #00BB00;">YES</span> </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; Fit via TL           :</span> <span style="color: #00BB00;">YES</span> </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; REVOLVER clustering  :</span> <span style="color: #00BB00;">YES</span> </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; Jackknife statistics :</span> <span style="color: #00BB00;">YES</span> </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; </span> </span>
<span class="co">#&gt; For summary statistics see `?Stats_*(x)` with * = {cohort, drivers, trees, fits, clusters, ...} </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; </span></span>
<span class="co">#&gt; <span style="color: #000000;">┌───────────────────────────────────────────────────────────────────────────────────────────┐</span></span>
<span class="co">#&gt; <span style="color: #000000;">│</span><span style="background-color: #FFAF00;">                                                                                           </span><span style="color: #000000;">│</span></span>
<span class="co">#&gt; <span style="color: #000000;">│</span><span style="background-color: #FFAF00;">   </span><span style="color: #000000; background-color: #FFAF00;">WARNING - Some patients have only one clone with drivers; they will just be expanded.</span><span style="background-color: #FFAF00;">   </span><span style="color: #000000;">│</span></span>
<span class="co">#&gt; <span style="color: #000000;">│</span><span style="background-color: #FFAF00;">                                                                                           </span><span style="color: #000000;">│</span></span>
<span class="co">#&gt; <span style="color: #000000;">└───────────────────────────────────────────────────────────────────────────────────────────┘</span></span>
<span class="co">#&gt; <span style="color: #949494;"># A tibble: 54 × 7</span></span>
<span class="co">#&gt;    patientID numBiopsies numMutations numDriverMutations numClonesWithDriver</span>
<span class="co">#&gt;    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>           <span style="color: #949494; font-style: italic;">&lt;int&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;int&gt;</span>              <span style="color: #949494; font-style: italic;">&lt;int&gt;</span>               <span style="color: #949494; font-style: italic;">&lt;int&gt;</span></span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span> CRUK0007            2            3                  3                   1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span> CRUK0010            2            3                  3                   1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span> CRUK0012            2            1                  1                   1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span> CRUK0018            4            4                  4                   1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span> CRUK0019            2            1                  1                   1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span> CRUK0021            2            4                  4                   1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span> CRUK0025            3            3                  3                   1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span> CRUK0026            2            4                  4                   1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span> CRUK0028            2            2                  2                   1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;">10</span> CRUK0029            6            4                  4                   1</span>
<span class="co">#&gt; <span style="color: #949494;"># … with 44 more rows, and 2 more variables: numTruncalMutations &lt;int&gt;,</span></span>
<span class="co">#&gt; <span style="color: #949494;">#   numSubclonalMutations &lt;int&gt;</span></span>

<span class="co"># Plot</span>
<span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">TRACERx_NEJM_2017_REVOLVER</span><span class="op">)</span></code></pre></div>
<p><img src="Input_formats_files/figure-html/unnamed-chunk-4-1.png" width="700"></p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by <a href="https://www.caravagnalab.org/" class="external-link">Giulio Caravagna</a>.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.2.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
